assets/config/0.5.1-coins.json

```diff
@@ -5266,6 +5266,25 @@
     "active": false,
     "currently_enabled": false
   },
+  "USBL": {
+    "active": true,
+    "coin": "USBL",
+    "name": "Balanced",
+    "coingecko_id": "balanced",
+    "coinpaprika_id": "balanced-balanced",
+    "currently_enabled": true,
+    "electrum": [
+      {
+        "url": "",
```

Comment:
A coin can not be added without an electrum, or else transactions / balances will not be functional.

Reply:
electrum urls updated with pull #323